metrics-operator/converter/sli_converter.md

```diff
@@ -0,0 +1,77 @@
+# SLI Converter
+
+## Description
+
+SLI converter is a tool to convert `sli.yaml` files used in [KeptnV1](https://v1.keptn.sh/) into the new
+`AnalysisValueTemplate` resources used in the new kubernetes-native [Keptn](https://lifecycle.keptn.sh/).
```

Comment:
I won't use the v1 links since they will soon disappear 
```suggestion
SLI converter is a tool to convert the old `sli.yaml` [file format](https://github.com/keptn/spec/blob/master/service_level_indicator.md) into the new
`AnalysisValueTemplate` [custom resource definition](../../docs/content/en/docs/crd-ref/metrics/v1alpha3/_index.md).
```